features/users/github_commit_count.feature

```diff
@@ -19,7 +19,7 @@ Feature: Displaying GitHub contribution statistics for user
   Scenario: Displays commit counts on user profile page
     Given I am on "profile" page for user "Bryan"
     Then I should see "Contributions"
-    And I should see "WebsiteOne - 388"
+    And I should see "WebsiteOne - 394"
```

Comment:
@diraulo right, I guess the specs are wrapped in VCR, but not the features. I'll open a ticket for that --> #812